global.d.ts

```diff
@@ -1,5 +1,10 @@
 declare const SERVICE_WORKER_CHECKSUM: string
 
 declare module '@bundled-es-modules/cookie' {
-  export * from 'cookie'
+  export * as default from 'cookie'
+}
+
+declare module '@bundled-es-modules/statuses' {
+  const source_default: any
```

Comment:
Maybe let's annotate this export? Just to be safe.

Reply:
I will add these type definitions, we're already depending on `@types/statuses`.